mwdb/web/src/commons/ui/ErrorBoundary.jsx

```diff
@@ -23,16 +23,29 @@ export function getErrorMessage(error) {
 export function Alert(props) {
     if (props.error) {
         return (
-            <div className="alert alert-danger">
-                {getErrorMessage(props.error)}
-            </div>
+            <AlertMessage
+                type={"danger"}
+                message={getErrorMessage(props.error)}
+            />
         );
-    } else if (props.success) {
-        return <div className="alert alert-success">{props.success}</div>;
-    } else if (props.warning) {
-        return <div className="alert alert-warning">{props.warning}</div>;
     }
-    return <div />;
+    if (props.success) {
+        return <AlertMessage type={"success"} message={props.success} />;
+    }
+    if (props.warning) {
+        return <AlertMessage type={"warning"} message={props.warning} />;
+    }
+    return <></>;
+}
+
+function AlertMessage(props) {
+    const { type, message } = props;
+
+    return (
+        <div className={`alert alert-${type}`}>
+            <span>{message}</span>
+        </div>
+    );
 }
 
```

Comment:
Is this part still needed?

Reply:
I found 3 more place where component `Alert` was still in use. It will be fixed/change/removed.